code/modules/surgery/organs/vocal_cords.dm

```diff
@@ -471,14 +471,6 @@
 			var/mob/living/L = V
 			L.emote("flip")
 
-	//SPEAK
-	else if((findtext(message, speak_words)))
-		cooldown = COOLDOWN_MEME
-		for(var/V in listeners)
-			var/mob/living/L = V
-			addtimer(CALLBACK(L, /atom/movable/proc/say, pick_list_replacements(BRAIN_DAMAGE_FILE, "brain_damage")), 5 * i)
```

Comment:
i'd be really happy if you could generate a random sentence in some other way instead of removing the command altogether